commands/utils/testsutils.go

```diff
@@ -38,8 +38,16 @@ func PrepareTestEnvironment(t *testing.T, projectName, testDir string) (string,
 	assert.NoError(t, err)
 	err = fileutils.CopyDir(filepath.Join("testdata", testDir), tmpDir, true, []string{})
 	assert.NoError(t, err)
-
-	restoreDir, err := Chdir(filepath.Join(tmpDir, projectName))
+	// Copy Git folder as .git if exists
+	currentDir := filepath.Join(tmpDir, projectName)
+	gitPath := filepath.Join(currentDir, "git")
+	if exists, err := fileutils.IsDirExists(gitPath, false); exists && err == nil {
```

Comment:
It looks like we're not asserting the error returned by the `IsDirExists` function.